src/mrack/providers/openstack.py

```diff
@@ -188,7 +188,10 @@ def _aggregate_networks(self, hosts):
                 continue
             network_type = host.get("network")
             if not self._is_network_type(network_type):
-                continue
+                raise ValidationError(
```

Comment:
Don't remember it precisely from top of my head and would need to read the whole worfklow, but shouldn't we expect that also a specific network can be set here?  I.e. not only network type.

Reply:
I believe it is set by `networks:` key I can recheck, i thought the same...

Reply:
https://github.com/neoave/mrack/blob/cdb86cba3657bb0fa27d4b212f379d45af1a7447/src/mrack/providers/openstack.py#L184